core/src/checks/check_sign_tx.c

```diff
@@ -126,6 +133,62 @@ int verify_sign_tx(void) {
     print_bytes(resp.signatures[0].der.bytes, resp.signatures[0].der.size);
     return -1;
   }
+  
   INFO("sign_tx passed");
   return 0;
 }
+/**
+ * This function does not make an effort to zeroize
+ * anything as all of the values are test values.
+ * It will just test our verify signature wrapper.
+ */
+
+int verify_check_signature_pub(void){
```

Comment:
I like this round trip test! Do you think we can, in addition, hardcode a signature value?